static/js/theme-map/Maps/PanTriggers.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Describes the possible of triggers for a map pan
+ *
+ * @enum {string}
+ */
+export default {
+  /**
+   * Indicates that the panTrigger is not set, and therefore there is no specific
+   * behavior that should occur if the panHandler is called with this PanTrigger.
+   * 
+   * This should be set if a user moves the map, or if a user clicks on a cluster.
```

Comment:
I think the terminology here is a bit confusing. If a user moves the map, that sounds to me like they did "pan" the map. Does pan have a specific definition in the context of WCAG?

Reply:
I'm using the word 'pan' in the same way that Google Maps and MapBox use it, which means that the map moved. Would it help if I removed the second sentence here?

Reply:
I guess the thing that's tripping me up is that if a user moves the map, it leads to a trigger of `UNSET`. In that case, it seems like the user took a manual action that triggered a pan of the map, almost as if the trigger should be `MANUAL` or something.

Reply:
My thinking here is that "API" is the only trigger that we care about for this item, so it felt unnecessary to set a "USER" trigger, however I could add one to match the ZoomTriggers (which have API, USER, and UNSET)

Reply:
Update: just added PanTriggers.User which behaves the same as the User ZoomTrigger